system-int-test/src/cmr/system_int_test/data2/opendata.clj

```diff
@@ -153,7 +153,11 @@
   (update-in opendata-results-map [:results :dataset]
              (fn [dataset]
                (into #{} (for [field dataset]
-                           (util/map-values #(if (sequential? %) (set %) %) field))))))
```

Comment:
All these lines are a little long (require scrolling). We usually set a ruler in our code editor to 100 to know when to split lines.